cmd/nerdctl/run_linux.go

```diff
@@ -108,8 +108,17 @@ func setPlatformOptions(opts []oci.SpecOpts, cmd *cobra.Command, id string) ([]o
 	if err != nil {
 		return nil, err
 	}
+
 	if privileged {
 		opts = append(opts, privilegedOpts...)
+
+		withoutHostDevices, err := cmd.Flags().GetBool("privileged-without-host-devices")
+		if err != nil {
+			return nil, err
+		}
+		if !withoutHostDevices {
+			opts = append(opts, oci.WithHostDevices, oci.WithAllDevicesAllowed)
```

Comment:
nit: can we keep it as two as `deviceOpts` together in run_security_linux.go

Reply:
A new struct `privilegedWithoutDevicesOpts` added for the new options after the original `privilegedOpts`